playbooks/site.yaml

```diff
@@ -28,6 +28,7 @@
 - import_playbook: zuul-merger.yaml
 - import_playbook: zuul-web.yaml
 - import_playbook: zuul-log-server.yaml
+- import_playbook: ha_log_cfg_sync.yaml
```

Comment:
Do we have to add a condition like ha_enabled? If it's not ha deployment, is it ok to import the playbook?

Reply:
if not a ha deployment, there should have host groups: "logserver-master", "zuul-scheduler-master", "zuul-scheduler-slave"... so the playbook will not actually run in non-ha deployment

Reply:
ok, I see.